.github/github_token_rotation.md

```
@@ -0,0 +1,63 @@
+    Update the cron expression under the schedule section in the YAML file according to your preferred frequency.
+
+Detailed Variables Description
+GITHUB_TOKEN_ROTATION_ROLE_TO_ASSUME_TEST - The AWS account role arn you will assume for token rotation in prod or test environment.
+
+AWS_REGION - The AWS region where the Secrets Manager is configured. Example: us-west-2.
+
+VILNACRM_APP_ID - The unique identifier for your GitHub App. Found in the GitHub App's settings.
+
+VILNACRM_APP_PRIVATE_KEY - The private key used to authenticate as the GitHub App. Generate this key in the GitHub App settings.
+
+SECRET_NAME - The name of the secret in AWS Secrets Manager where the GitHub token will be stored. Default is github-token.
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)